### PR TITLE
Map full name and locale date

### LINKED ISSUE #2365: Datetime format
**Describe the bug**
Datetime format is `Www Mmm dd hh:mm:ss yyyy` independent from locale (currently `asctime` is used).

**Expected behavior**
Date should follow system-format.

**Screenshots**
grafik

### PR DESCRIPTION
**Pull request by Laserlicht**
Shows the full path to the map with file extension.

Also uses system date&time output format now. Fixes #2365

### CODE REVIEW

File: lib/mapping/CMapInfo.h [hunk 1]
```diff
@@ -61,6 +62,7 @@ class DLL_LINKAGE CMapInfo
 		h & campaign;
 		h & scenarioOptionsOfSave;
 		h & fileURI;
+		h & fullFileURI;
```
Comment (Laserlicht) [inline lib/mapping/CMapInfo.h:65]: @IvanSavenko Is it correct to add the var here?

Comment (IvanSavenko) [reply lib/mapping/CMapInfo.h:65]: Yes, however this affects save format so please also:
- bump SERIALIZATION_VERSION by one (in CSerializer.h)
- add `if (version >= XXX)` check here (where XXX is new value of serialization version)

Comment (Laserlicht) [reply lib/mapping/CMapInfo.h:65]: The variable does not actually have to be in the savefile. Can I omit it then?

**Comment by Laserlicht** [comment]
included in #2517

**Pull request closed without merging**